src/components/Link.js

```diff
@@ -46,6 +46,22 @@ const InternalLink = styled(IntlLink)`
   }
 `
 
+const ExplicitLangInternalLink = styled(GatsbyLink)`
+  .is-glossary {
+    white-space: nowrap;
+  }
+  &.active {
+    color: ${(props) => props.theme.colors.primary};
+  }
+  &:hover {
+    svg {
+      fill: ${(props) => props.theme.colors.primary};
+      transition: transform 0.1s;
+      transform: scale(1.2);
+    }
+  }
```

Comment:
Also not sure we need this `&:hover` styling for the issue being reported. Don't believe there are any SVG's that are affected by the explicit english paths. Correct me if you find something, but if not I think this could go.

This also touches on your comment about DRY, and I think if we do reduce the above down to simply the ` &.active {...}` pseudo-selector then this isn't really a problem, and should still fix the reported issue.

Reply:
Because of how the links were abstracted I kept both for for coverage but to be honest I wasn't sure either way. 

Now that I think about it a missing glossary or hover probably would've been picked up quicker than a missing active state. Amending now 😃